File: 1º Semestre/Programação de Computadores/Lista de Exercicios 3/Python/11.py
"""
11. Elabore um algoritmo que leia um número de entrada que indicará a quantidade de 
números a serem Lidos. Em seguida, Leia n números (conforme o valor informado 
anteriormente) e imprima a soma e a média aritmética dos números informados.
"""

repInput = int(input ("Digite o numero de repetições: "))
userInput = int(input("Digite um numero: "))
soma = 0

for i in range(0, repInput):
    soma = soma + userInput

print("A soma é de", userInput,",",  repInput, "vezes, é igual a", soma)
print("A media aritmetica de", userInput, ",", repInput, "vezes, é igual a", soma/repInput)

File: 1º Semestre/Programação de Computadores/Lista de Exercicios 3/C-C++/11.cpp
#include <iostream>
using namespace std;

/*
11. Elabore um algoritmo que leia um número de entrada que indicará a quantidade de
números a serem lidos. Em seguida, leia n números (conforme o valor informado
anteriormente) e imprima a soma e a média aritmética dos números informados.
*/

int main(){
	int n, soma, media, numeros;
	
	cout << "Digite o numero de entradas: ";
	cin >> n;
	
	for(int i = 1; i <= n; i++){
		cout << "Digite um numero: ";
		cin >> numeros;
		
		soma += numeros;
		media = soma / n;
	}
	
	cout << "A soma dos numeros eh igual a: " << soma << "\n";
	cout << "A media aritmetica dos numeros eh igual a: " << media;
}
